plugins/outputs/pagerduty/pagerduty.go

```diff
@@ -0,0 +1,114 @@
+package pagerduty
+
+import (
+	"fmt"
+	pd "github.com/PagerDuty/go-pagerduty"
```

Comment:
we might not need to bring in this dependency. Since all we're using is "CreateEvent", which is a fairly simple http POST request.

I would recommend simple copying Pager Duty's code for that api call and keeping it in-repo: https://github.com/PagerDuty/go-pagerduty/blob/master/event.go

(you can put a comment with a link to the license at the top of the file)